### ISSUE TITLE
Support more connected wallets of the same type

### ISSUE THREAD

**Issue by prusnak** [wizardsardine/async-hwi#24]
Looking into `examples/hwi.rs` it seems that only one connected wallet of a certain type is supported.

What needs to be done so the lib supports multiple connected devices of the same type?

**Comment by darosior** [comment]
Yes, it's one of our priorities for the next version of Liana (see https://github.com/wizardsardine/liana/issues/165). @edouardparis should tackle it this week.

**Comment by darosior** [comment]
@edouardparis i think this was addressed, can you confirm?

**Comment by darosior** [comment]
Actually it's not entirely supported in Liana yet @kloaec reports.

**Comment by pythcoiner** [comment]
@edouardparis , looks like this has been addressed (https://twitter.com/KLoaec/status/1717491256664658297?t=6A-pyh8H5SxY_8XS98jRWw&s=19), right?
